docs/azure_jumpstart_arcbox/_index.md

```diff
@@ -0,0 +1,188 @@
+
+    ```shell
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+
+    For example:
+
+    ```shell
+    az ad sp create-for-rbac -n "http://AzureArcBox" --role contributor
+    ```
+
+    Output should look like this:
+
+    ```json
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcBox",
+    "name": "http://AzureArcBox",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+    > **Note: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest)**
+
+* Login to AZ CLI using the ```az login``` command.
+
+* Edit the [azuredeploy.parameters.json](../../azure_jumpstart_arcbox/azuredeploy.parameters.json) ARM template parameters file and supply some values for your environment.
+
+  * *aksDnsPrefix* - AKS unique DNS prefix
+  * *sshRSAPublicKey* - Your SSH public key
+  * *spnClientId* - Your Azure service principal id
+  * *spnClientSecret* - Your Azure service principal secret
+  * *spnTenantId* - Your Azure tenant id
+  * *windowsAdminUsername* - Client Windows VM Administrator name
+  * *windowsAdminPassword* - Client Windows VM administrator password
+  * *myIpAddress* - Your local IP address. This is used to allow remote RDP and SSH connections to the Client Windows VM and K3s Rancher VM.
+  * *logAnalyticsWorkspaceName* - Unique name for the ArcBox log analytics workspace
+
+    ![Screenshot showing example parameters](./parameters.png)
+
+* Now you will deploy the ARM template. Navigate to the local cloned [deployment folder](../../azure_jumpstart_arcbox) and run the below command:
+
+  ```shell
+  az group create --name <Name of the Azure resource group> --location <Azure Region>
+  az deployment group create \
+  --resource-group <Name of the Azure resource group> \
+  --template-file azuredeploy.json \
+  --parameters azuredeploy.parameters.json 
+  ```
+
+  ![Screenshot showing az group create](./azgroupcreate.png)
+
+  ![Screenshot showing az deployment group create](./azdeploy.png)
+
+* After deployment, you should see the ArcBox resources inside your resource group.
+
+  ![Screenshot showing az deployment group create](./deployedresources.png)
+
+* Open a remote desktop connection into _ArcBox-Client_. Upon logging in, multiple automated scripts will open and start running. These scripts will take 10-20 minutes to finish and once completed, Azure Data Studio will start. At this point, the deployment is complete.
+
+  ![Screenshot showing ArcBox-Client](./clientscript.png)
+
+  ![Screenshot showing ArcBox resources in Azure Portal](./rgarc.png)
+
+## Using ArcBox
```

Comment:
We need more explanations/details here on the tools installed on the client VM:

1. How does a user log in to the nested VMs? we need to provide the username/password in the guide.
2. How does a user switch between the AKS/k3s clusters using kubectx.
3. Show how to do _azdata login_ and the fact the CLI is there.

Without knowing these things they will have to dig in the scripts to figure it out.